mindmeld/auto_annotator.py

```diff
@@ -401,59 +389,85 @@ def parse(self, sentence, **kwargs):
 
 
 class SpacyAnnotator(Annotator):
-    """ (English) Annotator class that uses spacy to generate annotations.
-    Supported entities include: "sys_time", "sys_interval", "sys_duration", "sys_number",
-    "sys_amount-of-money", "sys_distance", "sys_weight", "sys_ordinal", "sys_quantity",
-    "sys_percent", "sys_org", "sys_loc", "sys_person", "sys_gpe", "sys_norp", "sys_fac",
-    "sys_product", "sys_event", "sys_law", "sys_langauge", "sys_work-of-art", "sys_other-quantity",
+    """Annotator class that uses spacy to generate annotations.
+    Depending on the language, supported entities can include: "sys_time", "sys_interval",
+    "sys_duration", "sys_number", "sys_amount-of-money", "sys_distance", "sys_weight",
+    "sys_ordinal", "sys_quantity", "sys_percent", "sys_org", "sys_loc", "sys_person",
+    "sys_gpe", "sys_norp", "sys_fac", "sys_product", "sys_event", "sys_law", "sys_langauge",
+    "sys_work-of-art", "sys_other-quantity".
     For more information on the supported entities for the Spacy Annotator check the MindMeld docs.
     """
 
-    def __init__(self, app_path, config=None):
-        """ Initializes an annotator.
+    def __init__(
+        self,
+        app_path,
+        annotation_rules=None,
+        language=None,
+        locale=None,
+        overwrite=False,
+        spacy_model_size="lg",
+        unannotate_supported_entities_only=True,
+        unannotation_rules=None,
+    ):
+        """Initializes a SpacyAnnotator.
 
         Args:
-            app_path (str): The location of the MindMeld app
-            config (dict, optional): A config object to use. This will
-                override the config specified by the app's config.py file.
+            app_path (str): The location of the MindMeld app.
+            annotation_rules (list): List of Annotation rules.
+            language (str, optional): Language as specified using a 639-1/2 code.
+            locale (str, optional): The locale representing the ISO 639-1 language code and \
+                ISO3166 alpha 2 country code separated by an underscore character.
+            overwrite (bool): Whether to overwrite existing annotations with conflicting spans.
+            spacy_model_size (str, optional): Size of the Spacy model to use. ("sm", "md", or "lg")
+            unannotate_supported_entities_only (bool): Only allow removal of supported entities.
+            unannotation_rules (list): List of Annotation rules.
         """
-        super().__init__(app_path=app_path, config=config)
+        super().__init__(
+            app_path,
+            annotation_rules=annotation_rules,
+            language=language,
+            locale=locale,
+            overwrite=overwrite,
+            unannotate_supported_entities_only=unannotate_supported_entities_only,
+            unannotation_rules=unannotation_rules,
+        )
```

Comment:
You could do `super.__init__(*args, **kwargs)` after removing `spacy_model_size ` from it.